src/lib/toolchain.rs

```diff
@@ -8,12 +8,17 @@
 mod toolchain_test;
 
 use crate::types::CommandSpec;
+use std::process::{Command, Stdio};
 
 pub(crate) fn wrap_command(
     toolchain: &str,
     command: &str,
     args: &Option<Vec<String>>,
 ) -> CommandSpec {
+    if !has_toolchain(toolchain) {
+        error!("Please install the toolchain {} using rustup before building!", &toolchain);
```

Comment:
maybe people are not building anything but using this as a simple task runner.
maybe rephrase to: "Error: missing toolchain: {}" which is really simple message.

Reply:
Fixed :+1: